### PR TITLE
Implements --get, --set, and --remove for `env`

### PR DESCRIPTION
**Pull request by pwoolcoc**

### CODE REVIEW

**Comment by jonathandturner** [comment]
First off, thanks for this. Much appreciated.

I hate to say it, but this functionality is already in Nu, though it's a bit harder to find.  You can update the environment variables using the technique in https://www.nushell.sh/blog/2019/11/05/nushell-0_5_0.html#adding-variables-to-your-environment.  It's a bit wordy now, but we'll be adding aliases soon that should help make it quite a bit smaller.

One thing that will probably fall out from that is moving away from the `env` command towards using other techniques.

**Comment by pwoolcoc** [comment]
@jonathandturner ahh I can't believe I missed that! thanks for the heads-up, i'll be sure to check the docs better next time!
